daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/crypto/Hash.scala

```diff
@@ -355,4 +356,13 @@ object Hash {
       .addStringSet(parties)
       .build
 
+  // For Corda
```

Comment:
I would recommend rephrasing this to clarify _why_ this enables DAML-on-Corda rather than saying that it does.